charts/cinder-csi-plugin/templates/controllerplugin-deployment.yaml

```diff
@@ -45,6 +43,7 @@ spec:
           args:
             - "--csi-address=$(ADDRESS)"
             - "--timeout={{ .Values.timeout }}"
+            - "--leader-election=true"
```

Comment:
I see that enabling leader election might require additional rbac , are they inplace already?

Reply:
Yes actually there is a `external-resizer-cfg` Role with correspoding rbac, however as it was not only related to resizer. As all the roles refrence one `sa`, even that was enough. However for better consistency, I refactored it and added the role to each of the 4 components.